src/components/ClusterTemplateQuotas/NewQuotaDialog/newQuotaValidationSchema.ts

```diff
@@ -14,30 +14,30 @@ const useNewQuotaValidationSchema = (clusterTemplateCost: number) => {
   //It's used for testing unique names, if it fails or not loaded yet, the backend will block the creation
   const [allQuotas] = useAllQuotas();
   const usedQuotaNames = React.useMemo(
-    () => allQuotas.map((quota) => quota.metadata?.name),
+    () => allQuotas.map((quota) => quota.metadata?.name).filter(Boolean) as string[],
```

Comment:
`Boolean` is quite resource heavy. Lets just use `reduce`

Reply:
Done